benchmarks/python/test_rope.py

```diff
@@ -52,12 +57,17 @@ def fwd_call(inp):
         "hf_qwen2",
         "hf_phi3",
         "hf_mistral_nemo",
+        "litgpt-gemma-2-9b",
+        "litgpt-mistral-7b",
+        "litgpt-meta-llama-3-8B",
+        "litgpt-phi3.5-mini",
```

Comment:
@xwang233 do we need to manually add new entries in dashboard?

Reply:
it might not show up in the PR benchmark results, but it will show up in nightly benchmark results once merged